sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetQuerySuite.scala

```diff
@@ -40,7 +42,8 @@ import org.apache.spark.util.Utils
 /**
  * A test suite that tests various Parquet queries.
  */
-abstract class ParquetQuerySuite extends QueryTest with ParquetTest with SharedSparkSession {
+abstract class ParquetQuerySuite extends QueryTest with ParquetTest with SharedSparkSession
+  with ExplainSuiteHelper{
```

Comment:
Could you make a new test suite for these 447 line test code, @huaxingao ?

Reply:
Done

Reply:
This change can be reverted now?